setup.py

```diff
@@ -106,6 +107,7 @@ def read(fname):
                 'compile_translations': compile_translations },
       entry_points={'console_scripts': [
           'webant=webant.webant:main',
+          'webant-manage=webant.manage:main',
```

Comment:
why did you choose this name?
the application name is libreant. The web interface module name is webant.
It's very strange that the command to menage libreantdb it's called webant-manage.